packages/medusa/src/services/event-bus.ts

```diff
@@ -55,28 +55,28 @@ export default class EventBusService {
     config: ConfigModule,
     singleton = true
   ) {
-    const opts = {
-      createClient: (type: string): Redis.Redis => {
-        switch (type) {
-          case "client":
-            return redisClient
-          case "subscriber":
-            return redisSubscriber
-          default:
-            if (config.projectConfig.redis_url) {
-              return new Redis(config.projectConfig.redis_url)
-            }
-            return redisClient
-        }
-      },
-    }
-
     this.config_ = config
     this.manager_ = manager
     this.logger_ = logger
     this.stagedJobRepository_ = stagedJobRepository
 
     if (singleton) {
+      const opts = {
```

Comment:
comment: Just a note, I think we should be careful about including such changes in a completely unrelated PR. It will be harder to revert in case we experience issues (although unlikely to happen in this case), but I just wanted to mention it. Flipping the verbosity flag is a somewhat similar change.

Reply:
I think it's fine to keep them in, but I just wanted to add it as a note for us to be aware of it going forward 👍